docs/src/api/class-browsercontext.md

```diff
@@ -10,6 +10,8 @@ context.
 Playwright allows creating "incognito" browser contexts with [`method: Browser.newContext`] method. "Incognito" browser
 contexts don't write any browsing data to disk.
 
+:::note Attention! There would be memory consuming/leak around 150 MiB per 1 hour when using same context for a long time for operations page.goto(). Closing context would free memory. :::
```

Comment:
That might be misleading - it can be 0 or it can be 150Mb per second, depending on the page. Also technically it is not a leak. What we should say is that context should be created per unit of work, because it retains information about the pages.

Reply:
During my test run I get up to ~1.6GB after 10 minutes, then 2.5GB after another 10 minutes.